bears/vcs/git/GitConflictMarkerBear.py

```diff
@@ -0,0 +1,35 @@
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.results.Diff import Diff
+from coalib.results.SourceRange import SourceRange
+
+
+class GitConflictBear(LocalBear):
+    def run(self, filename, file):
+        marker_free_code = []
+
+        merge_conflict_starts = [line_number for line_number, line in
+                                 enumerate(file) if
+                                 line.startswith('<<<<<<< ')]
+
+        merge_conflict_split = [line_number for line_number, line in
+                                enumerate(file) if
+                                line.startswith('=========')]
+
+        merge_conflict_ends = [line_number for line_number, line in
+                               enumerate(file) if
+                               line.startswith('>>>>>>> ')]
+
+        for line_number, line in enumerate(file):
+            if (line_number not in merge_conflict_starts and
+                line_number not in merge_conflict_split and
+                    line_number not in merge_conflict_ends):
+                marker_free_code.append(line)
+
+        diffs = Diff.from_string_arrays(file, marker_free_code).split_diff()
```

Comment:
This approach might fail if the markers are beneath each other or if the default split-width for `split_diff()` is changed. I would rather recommend creating tuples with (marker-start-pos, split-pos, end-pos) and iterate over those

Reply:
should be possible with `list(zip(merge_conflict_starts, merge_conflict_split, merge_conflict_ends))`

Reply:
I don't see how iterating over tuples would be any different can you please elaborate?

Reply:
ping @Makman2